packages/ui/ui-theme/src/styles/layers/scrollbar.css

```
@@ -12,6 +12,7 @@
   }
   .dx-scrollbar-thin::-webkit-scrollbar-thumb {
     @apply bg-transparent;
+    transition: background 0.15s;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)